### PR TITLE
added tooltip.js and css styles

### PR DESCRIPTION
**Pull request by BratCat**
added tooltip.js to update_status.html and tooltip styling to _buttons.scss

### CODE REVIEW

File: client/templates/status/update-status.html [hunk 1]
```diff
@@ -5,12 +5,15 @@
...
+        <button class="btn btn-circle btn-warning btn-outline btn-hangout-status btn-tooltip" data-type="teaching" title="Create a teaching hangout to practice teaching a concept or to walk participants through a codebase. This is your chance to share what you know!" data-toggle="tooltip" data-placement="top"><i class="fa fa-user"></i></button>
+        <button class="btn btn-circle btn-success btn-outline btn-hangout-status btn-tooltip" data-type="collaboration" title="Create a collaboration hangout to pair program on a coding exercise, talk through a chapter of a tutorial, or work on an open-sourced project together." data-toggle="tooltip" data-placement="top"><i class="fa fa-users"></i></button>
         <!--Character counter for 140 character limit-->
         <span class="charactersLeft">{{characterCount}}</span>
         {{/if}}
         <button id="update-{{type}}-btn" class="btn btn-cb2 pull-right">{{_ "update"}}</button>
     </div>
+    <script>
+   $(function () { $("[data-toggle = 'tooltip']").tooltip(); });
```
Comment (distalx) [inline client/templates/status/update-status.html]: jQuery code should be applied inside a `Template helper` or `Template onRendered` function.
http://blazejs.org/api/templates.html#Blaze-TemplateInstance-$

Comment (BratCat) [inline client/templates/status/update-status.html]: Ok I'll get that changed. Sorry about that!

Comment (lpatmo) [inline client/templates/status/update-status.html]: Thanks @BratCat!

Comment (BratCat) [inline client/templates/status/update-status.html]: I moved the JQuery to the JS file, took some hunting on that one. It didn't want to work. Updated html file with classes and tooltip info. Created new .scss file for styles and added the @import to the styles.scss file.

**Comment by BratCat** [comment]
@ camilleriluke Thanks! Glad I could help!

**Comment by lpatmo** [comment]
Looks great, @BratCat! 

I resolved a merge conflict introduced by the merging of another pull request and slightly changed the text on the hangout button status area, then merged in the branch. 

Thanks so much for your work on this issue! :)

**Pull request merged**